cypress/e2e/HANKA.cy.js

```diff
@@ -0,0 +1,265 @@
+ /// <reference types="cypress" />
+
+
+//describe('Login Standard User', function () {
+it('Open website, type and login as Standard User', function() {    //user
+  cy.visit('/')
+
+  cy.get('[data-test="username"]')
+    .type('standard_user')
+    .should('have.value', 'standard_user')
+  cy.get('[data-test="password"]')
+    .type('secret_sauce')
+    .should('have.value', 'secret_sauce')
+  cy.get('[data-test="login-button"]')  
+    .click()  
+    .title().should('include', 'Swag Labs')
```

Comment:
This assertion will pass no matter where the user will be navigated to after login. The page title is the same for every sub-page. 
Please change the assertion to one verifying if the URL contains `/inventory.html` or verify if the it show this `Products` message
![image](https://user-images.githubusercontent.com/76937912/179724273-b2f9df9d-738b-474f-8564-7cbb8102bb7d.png)